Userland/Applications/ThemeEditor/main.cpp

```diff
@@ -266,15 +276,22 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     auto file_menu = TRY(window->try_add_menu("&File"));
 
     auto update_window_title = [&] {
-        window->set_title(String::formatted("{} - Theme Editor", path.value_or("Untitled")));
+        window->set_title(String::formatted("{} [*] - Theme Editor", path.value_or("Untitled")));
```

Comment:
don’t put a space before the `[*]` mark. it always results in a double space after the filename

```suggestion
        window->set_title(String::formatted("{}[*] - Theme Editor", path.value_or("Untitled")));
```

Reply:
Thanks! I didn't notice that. :^)